### PR TITLE
[zh] Translate and sync KubeCon Europe 2023 Istio Day blog

### PR DESCRIPTION
**Pull request by wilsonwu**
Translate and sync: https://istio.io/latest/blog/2023/istio-at-kubecon-eu/ to Chinese.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+description: 快速回顾在阿姆斯特丹 RAI 的 KubeCon 欧洲站中的 Istio。
+publishdate: 2023-04-27
+attribution: "Faseela K，来自 Istio Day 计划委员会"
+keywords: [Istio Day,IstioCon,Istio,conference,KubeCon,CloudNativeCon]
+---
+
+4 月 18 日至 21 日，云原生及开源社区欢聚在举办于阿姆斯特丹的 2023 年第一场 KubeCon 中。
+这是一场由云原生计算基金会组织举办的为期四天的盛会，
+我们从只是作为 ServiceMeshCon 的参与者到现在举办第一个位于同一地点的官方项目活动，
+这对于 Istio 来说是特别的。
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:12]:
```suggestion
这是一场由云原生计算基金会组织且专为 Istio 举办的为期四天的盛会，
也是在这片土地上，我们从当年 ServiceMeshCon 的一个参与者演变为第一个官方项目活动 (Istio Day Europe) 的组织者。
```

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+{{< image width="40%"
+    link="./istio-day-welcome.jpg"
+    caption="欢迎来到 Istio Day 2023 年欧洲站"
+    >}}
+
+在两位程序委员会主席 Mitch Connors 和 Faseela K 的精彩开场主题演讲后，Istio Day 正式开始。
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:19]:
```suggestion
在两位技术程序委员会 (Program Committee) 主席 Mitch Connors 和 Faseela K 的精彩开场主题演讲后，Istio Day 正式开始。
```

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+本次活动涵盖了从新功能到最终用户话题等丰富的内容，大厅内始终人头攒动。
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:20]:
```suggestion
本次活动涵盖了从新功能到最终用户话题等丰富的内容，展厅内始终人头攒动。
```

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+[开场主题演讲](https://youtu.be/h9EgMrJ0ahs)以突击测试的形式利用 Istio 的一些有趣内容达到了破冰效果，
+并对我们的贡献者、维护者、发布经理以及用户的日常工作表示了认可。
+
+{{< image width="75%"
+    link="./opening-keynote.jpg"
+    caption="Istio Day 2023 欧洲站开场主题"
+    >}}
+
+随后是来自 TOC 成员 Lin Sun 和 Louis Ryan 的 2023 年 [路线图更新议题](https://youtu.be/GQccKyVe0R8)。
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:29]:
```suggestion
随后来自 TOC 的成员 Lin Sun 和 Louis Ryan 谈到了 2023 年[路线图更新议题](https://youtu.be/GQccKyVe0R8)。
```

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+接着由 Christian Posta 和 John Howard 带来了期待已久的 [Ambient Mesh 安全态势](https://youtu.be/QnfrbbY_Hy4) 议题，
+并在社区中引起了一些有趣的讨论。
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:31]:
```suggestion
接着由 Christian Posta 和 John Howard 带来了期待已久的、曾在社区中引起一些有趣讨论的
[Ambient Mesh 安全态势](https://youtu.be/QnfrbbY_Hy4)议题。
```

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+After this we stepped into our first [end user talk from John Keates from Wehkamp](https://youtu.be/Gb_I2RJr8kQ),
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:32]:
```suggestion
```

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+在此之后，我们进入了第一个来自荷兰本地公司 [Wehkamp 公司 John Keates 的最终用户访谈](https://youtu.be/Gb_I2RJr8kQ)，
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:33]:
```suggestion
在此之后，出场的是第一次来自荷兰本地公司 [Wehkamp 公司 John Keates 的最终用户访谈](https://youtu.be/Gb_I2RJr8kQ)，
```

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+紧跟着是来自彭博社讲师 Alexa Griffith 和 Zhenni Fu 带来的关于 [如何使用 Istio 保护他们高特权财务信息](https://youtu.be/f6jMix46ZD8) 的演讲。
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:34]:
```suggestion
紧跟着来自彭博社的讲师 Alexa Griffith 和 Zhenni Fu 演讲了[如何使用 Istio 保护他们的高特权财务信息](https://youtu.be/f6jMix46ZD8)。
```

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+当 Zack Butcher 谈到 [使用 Istio 实现控制合规性](https://youtu.be/gIntE4Nn5r4) 话题时，
+专注安全性这一点在 Istio Day 中得到了更多见证。
+We also had lightning talks covering [faster Istio development environments](https://youtu.be/Onsukvmmm50),
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:37]:
```suggestion
```

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+    >}}
+
+随后是来自 TOC 成员 Lin Sun 和 Louis Ryan 的 2023 年 [路线图更新议题](https://youtu.be/GQccKyVe0R8)。
+接着由 Christian Posta 和 John Howard 带来了期待已久的 [Ambient Mesh 安全态势](https://youtu.be/QnfrbbY_Hy4) 议题，
+并在社区中引起了一些有趣的讨论。
+After this we stepped into our first [end user talk from John Keates from Wehkamp](https://youtu.be/Gb_I2RJr8kQ),
+在此之后，我们进入了第一个来自荷兰本地公司 [Wehkamp 公司 John Keates 的最终用户访谈](https://youtu.be/Gb_I2RJr8kQ)，
+紧跟着是来自彭博社讲师 Alexa Griffith 和 Zhenni Fu 带来的关于 [如何使用 Istio 保护他们高特权财务信息](https://youtu.be/f6jMix46ZD8) 的演讲。
+当 Zack Butcher 谈到 [使用 Istio 实现控制合规性](https://youtu.be/gIntE4Nn5r4) 话题时，
+专注安全性这一点在 Istio Day 中得到了更多见证。
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:36]:
```suggestion
 Istio Day 见证了人们更加注重安全性，这一点在 Zack Butcher
 讲到[使用 Istio 实现控制合规性](https://youtu.be/gIntE4Nn5r4)话题时达到了高潮。
```

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+We also had lightning talks covering [faster Istio development environments](https://youtu.be/Onsukvmmm50),
+我们还就来自于 [更快速的 Istio 开发环境](https://youtu.be/Onsukvmmm50) 进行了闪电演讲，
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:38]:
```suggestion
```

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+我们还针对分别来自 Mitch Connors、Zhonghu Xu 和 Matt Turner 的 [更快的 Istio 开发环境](https://youtu.be/Onsukvmmm50)、
+[Istio 资源隔离指南](https://youtu.be/TmlfQjChmNU) 和 [混合云部署安全](https://youtu.be/xejbMNbOwXk) 话题进行了闪电演讲。
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:40]:
```suggestion
Mitch Connors、Zhonghu Xu 和 Matt Turner 分别就[更快速的 Istio 开发环境](https://youtu.be/Onsukvmmm50)、
[Istio 资源隔离指南](https://youtu.be/TmlfQjChmNU)和[混合云部署安全](https://youtu.be/xejbMNbOwXk)话题进行了闪电演讲。
```

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+
+{{< image width="75%"
+    link="./istioday-hall.jpg"
+    caption="Istio Day 2023 欧洲站人头攒动的议题"
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:44]:
```suggestion
    caption="Istio Day 2023 欧洲站热点议题"
```

**Review by windsonsea — COMMENTED** [review]
If you read it through again and again after submitting a PR, you can find much more to improve.

**Comment by wilsonwu** [comment]
@windsonsea Thanks for the carefully review :)

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 14]
```diff
@@ -0,0 +1,106 @@
...
+以下在 KubeCon 中的议题也都是基于 Istio 的，几乎所有议题都有大量的人参加：
+
+* [Istio 的未来 - Sidecar、无 Sidecar 还是两者兼而有之？](https://sched.co/1HySB)
+* [在生产环境中使用 ArgoCD 运行多租户 Istio](https://sched.co/1Hyd1)
+* [使用任意编程语言创建 Istio 过滤器](https://sched.co/1HybK)
+* [使用 Kubernetes 和服务网格的自动化云原生事件响应](https://sched.co/1HyZ9)
+* [使用无代理 gRPC 和 Istio 为有状态应用程序自动缩放弹性 Kubernetes 基础设施](https://sched.co/1HyXz)
+* [开发 Istio 的心智模型：从 Kubernetes 到 Sidecar 再到 Ambient](https://sched.co/1HyZj)
+* [服务网格的未来 - Sidecar、无 Sidecar 还是无代理模式？ - 小组讨论会](https://sched.co/1Hydb)
+* [十大 Istio 安全风险和缓解策略的](https://sched.co/1HyPQ)
```
Comment (wilsonwu) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:75]:
```suggestion
* [十大 Istio 安全风险和缓解策略](https://sched.co/1HyPQ)
```

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 15]
```diff
@@ -0,0 +1,106 @@
...
+* [使用任意编程语言创建 Istio 过滤器](https://sched.co/1HybK)
+* [使用 Kubernetes 和服务网格的自动化云原生事件响应](https://sched.co/1HyZ9)
+* [使用无代理 gRPC 和 Istio 为有状态应用程序自动缩放弹性 Kubernetes 基础设施](https://sched.co/1HyXz)
+* [开发 Istio 的心智模型：从 Kubernetes 到 Sidecar 再到 Ambient](https://sched.co/1HyZj)
+* [服务网格的未来 - Sidecar、无 Sidecar 还是无代理模式？ - 小组讨论会](https://sched.co/1Hydb)
+* [十大 Istio 安全风险和缓解策略](https://sched.co/1HyPQ)
+
+Istio 在 KubeCon 展厅中有一个全天候展台，
+很大一部分问题都是关于我们在 CNCF 毕业状态相关的。
+我们很高兴了解到我们的用户热切地等待我们毕业的消息，我们保证正在为此积极努力！
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:79]:
```suggestion
我们很高兴了解到用户热切期待我们毕业的消息，我们承诺正在为此积极努力！
```

File: content/zh/blog/2023/istio-at-kubecon-eu/index.md [hunk 16]
```diff
@@ -0,0 +1,106 @@
...
+
+我们的许多 TOC 成员和维护者也在展位上提供了支持，
+围绕 Istio Ambient Mesh 也进行了很多有趣的讨论。
+
+{{< image width="75%"
+    link="./toc-members-at-kiosk.jpg"
+    caption="KubeCon 欧洲站，Istio 展台的更多支持"
+    >}}
+
+另一个亮点是 Istio TOC 和指导成员以及作者 Lin Sun 和 Christian Posta 为
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-eu/index.md:94]:
```suggestion
另一个亮点是 Istio TOC 成员、指导委员会成员兼作者 Lin Sun 和 Christian Posta 为
```

**Review by windsonsea — COMMENTED** [review]
Thanks. Raised some more text to discuss here.

**Comment by windsonsea** [comment]
Thanks. LGTM.

**Comment by wilsonwu** [comment]
Thanks for these great reviews!

**Review by windsonsea — APPROVED** [review]

**Pull request merged**